### DISCUSSION TITLE
Real-Debrid Support F.A.Q.

### DISCUSSION THREAD

**Discussion by yowmamasita** [debridmediamanager/awesome-debrid#7] [General]
I spent some time scraping their support pages and formatted it in a single pager:

# Real-Debrid FAQ

## I would like to be refunded
If you did not use your account, you can get a refund up to 14 days after your payment in accordance with our Terms Of Services available at this address: [https://real-debrid.com/terms](https://real-debrid.com/terms)

Before contacting us, please go to this URL to check if your account has been used or not: [https://real-debrid.com/traffic](https://real-debrid.com/traffic)

## My account has not been credited after an Amazon Pay payment
First, please check that your payment has been charged on Amazon Pay. To this end, please go to: [https://pay.amazon.com/](https://pay.amazon.com/)

Most times, the payment is still under authorization process. For some credit cards, Amazon can take up to 6 hours to really debit your card.

Second, please check your account payments history on your Real-Debrid account. To this end, please go to: [https://real-debrid.com/payments-history](https://real-debrid.com/payments-history)

After checking, if you think that the payment has been charged and not credited on your Real-Debrid account, please contact us.

## My account has not been credited after a credit / debit card payment
A payment made on Real-Debrid appears as "REAL DEBRID" on your bank's statement.

In case of a refused payment (red box), some banks may display a line on your online statement but it is a pre-authorization, not a debit. Please wait a maximum of 10 business days to see it refunded. Note that our service does not have the funds linked to this transaction.

If you think that the payment has been charged and not credited on your Real-Debrid account, please contact us.

## My account has not been credited after a Bitcoin / Ethereum / Litecoin payment
A cryptocurrency payment can take quite some time to be confirmed and validated on Real-Debrid.

Please wait 2 hours before contacting us.

If after this time your payment still has not been validated, please contact us.

## My credit / debit card or Amazon Pay payment has been refused
First, please check if 3D Secure (Verified by Visa / MasterCard SecureCode) is enabled on your credit / debit card. 3D Secure aims to secure the payments you make on the internet.

Second, please ensure you can make oversea payments in euro (EUR - €) currency. Please contact your bank if you are not sure about this.

In case of a refused payment, some banks may display a debit on your bank statement (pre-authorization). Please wait a maximum of 10 business days to see it refunded. Note that our service does not have the funds linked to this transaction.

## I was charged twice when I paid by credit / debit card
While doing a credit / debit card payment, some banks may display an extra debit on your bank statement due to a pre-authorization (pending payment). Please wait a maximum of 10 business days to see it refunded.

If the second amount is not the subscription cost but a much lower fee, then it is probably your bank exchange rate fee (you are debited in € / EUR currency).

After checking, if you think that the payment has been charged twice, please contact us.

## I would like to pay with Paypal
Unfortunately, Paypal is not available on Real-Debrid. Please note that there are many other payment methods you can use:

- Credit / debit card
- Amazon Pay (Visa, MasterCard, American Express, Visa Electron, Delta, and Maestro)
- Prepaid Tickets (Paysafecard and Neosurf)
- Coinbase (Bitcoin and Ethereum)
- SMS / Phone (15 premium days offer only, not available for all countries)

Payment page: [https://real-debrid.com/premium](https://real-debrid.com/premium)

## I want to buy a voucher through an official Real-Debrid reseller
You can buy a voucher at one of our official resellers.

The list is here: [https://real-debrid.com/resellers](https://real-debrid.com/resellers)

## I want to use a voucher to get my premium days
To use your voucher, you need a Real-Debrid account and have to be logged into it. Once you are connected, please go to this address to validate your voucher: [https://real-debrid.com/voucher](https://real-debrid.com/voucher)

If you are getting "Your code is incorrect" error, please contact the reseller first. If you can't solve the issue with him (and only in this situation), you can contact us.

## I paid a voucher through a Real-Debrid reseller but I didn't get it
First, please try to contact the reseller directly. Note that not all resellers validate their orders automatically. It can take up to 48 opened hours.

If after this deadline, you still haven't received your voucher or got a reply from the reseller, please contact us.

## I want to see my payments history
First, you need to be logged into your Real-Debrid account. You can then find your payments history here: [https://real-debrid.com/payments-history](https://real-debrid.com/payments-history)

Please note that you can find this link in the "My account" page: [https://real-debrid.com/account](https://real-debrid.com/account)

## I didn't receive my Rentabiliweb code (SMS+ / Phone)
If you didn't receive your code after a SMS / phone payment, please contact Rentabiliweb directly:

- [https://payment.rentabiliweb.com/support/?docId=114075&siteId=390002&lang=en](https://payment.rentabiliweb.com/support/?docId=114075&siteId=390002&lang=en)

## I want to validate my Rentabiliweb code
First, you need to be logged into your Real-Debrid account.

To validate your Rentabiliweb code, go to this page: [https://real-debrid.com/offers-1](https://real-debrid.com/offers-1). Click on "SMS", paste, and validate your code in the provided field.

If you're getting "Your code is incorrect" error, you probably misnoted your code or waited too long to validate it (it is only valid for 24 hours). In case of any problem, please contact Rentabiliweb directly:

- [https://payment.rentabiliweb.com/support/?docId=114075&siteId=390002&lang=en](https://payment.rentabiliweb.com/support/?docId=114075&siteId=390002&lang=en)

## I would like to know how much a Real-Debrid account costs
You can find all of our premium offers here: [https://real-debrid.com/premium](https://real-debrid.com/premium)

All our prices are in EURO (EUR - €) but it is also displayed in your currency based on your current location. If you want to double-check, you can convert them to your currency with this website: [http://www.xe.com/](http://www.xe.com/)

Informations:
- Additional fees may be applied by your bank if your bank account is in another currency (for currency exchange).
- There are no recurring payments (one-time payment only).
- Days are cumulative, it means that if you still have remaining days, the days you buy will be added to your current number of days.

## I would like to know if payments are recurring
There are no recurring payments on Real-Debrid (one-time payment only).

Please note that we are not storing payment methods of our users. Therefore, we cannot receive any payment without an action from you. Do not hesitate to ask your family or anyone who can access your bank card if you do not recognize a debit from our company (Payments will appear as "REAL DEBRID" on your statements).

## I have no audio or video on files I play on my device
If the video you are playing on your device has no sound or no video, you are missing a Codec. The easier fix is to change the decoder of your player or use another player (VLC has a lot of built-in Codecs: [https://www.videolan.org/](https://www.videolan.org/))

This problem has nothing to do with our service.

For more details about codecs: [https://en.wikipedia.org/wiki/Codec](https://en.wikipedia.org/wiki/Codec)
To find out what are the audio / video Codecs of a file, "MediaInfo" software can help you: [https://mediaarea.net/en/MediaInfo](https://mediaarea.net/en/MediaInfo)

## How to authorize a device / application on my account (where is authorization code)
The Real-Debrid authorization code is given to you by the third-party application you are trying to connect your account to.

Authorization codes change every 2 minutes and are different for each application that you want to authorize. Therefore, they are not reusable or supplied after payment.

- To authorize a third-party application on your account, please go here: [https://real-debrid.com/device](https://real-debrid.com/device)
- To view or delete authorizations previously made on your account, please go here: [https://real-debrid.com/devices](https://real-debrid.com/devices)

## I can't authorize a device / application on my account (invalid / expired code error)
Authorization codes change every 2 minutes and are different for each application that you want to authorize. Therefore, they are not reusable or supplied after payment.

Please make sure that the third-party application you want to authorize is compatible with our service.

Our authorization codes are composed of numbers and letters in uppercase and are longer than 4 characters long. If you don't get such a

 code, it is probably because you have an authorization code for another service (Scroll to find ours).

## How devices / applications work with Real-Debrid
Once a third-party application is authorized to use your account, it sends to our service filelocker links (e.g., Rapidgator.net, Uploaded.net, etc.) or torrent files or magnet links.

- In case of torrent files or magnet links, our service converts them into a direct-download link (which can then be used by a player).
- In the case of filelocker links, our service removes Free-Mode restrictions (download speed, download volume, etc.).

**Important Information:**
You are the one feeding links / torrents to Real-Debrid (from scripts / third-party applications or web browser). In no way does Real-Debrid provide links, torrents, or magnet links to ANY content. We are not a content provider and will never be. You are legally responsible for links, torrents, or magnet links that you or a third-party application sends to our service.

## I would like to be an official Real-Debrid reseller
Unfortunately, we are not accepting more resellers at the moment.

## Which hosters / filelockers are supported and their status on Real-Debrid?
The hosters list we're supporting is available on the website footer. You can check which hosters / filelockers are supported on Real-Debrid and their status here (you must be logged in): [https://real-debrid.com/compare](https://real-debrid.com/compare)

- A green thumb is displayed: This hoster is available on Real-Debrid.
- A red thumb is displayed: The hoster is currently unavailable on Real-Debrid.
- "Not Supported" is displayed: This hoster is not supported on Real-Debrid.

## Where can I find extensions for web browsers?
We developed extensions for these web browsers:

- Chrome: [https://chrome.google.com/webstore/detail/real-debrid-extension/oefkkgfcahbeccgckjgbnfclcmnjgidg](https://chrome.google.com/webstore/detail/real-debrid-extension/oefkkgfcahbeccgckjgbnfclcmnjgidg)
- Firefox: [https://fcdn.real-debrid.com/0767/plugins/firefox@realdebrid-1.5.7_signed.xpi](https://fcdn.real-debrid.com/0767/plugins/firefox@realdebrid-1.5.7_signed.xpi)
- Opera: [https://addons.opera.com/en/extensions/details/real-debrid-extension/](https://addons.opera.com/en/extensions/details/real-debrid-extension/)
- Safari (version < 13): [https://safari-extensions.apple.com/details/?id=com.xtnetwork.safari.realdebrid-JN6WLL77A8](https://safari-extensions.apple.com/details/?id=com.xtnetwork.safari.realdebrid-JN6WLL77A8)

Once installed, the extension will automatically search inside pages you visit for hosters links that can be downloaded via our service. You will no longer need to go to Real-Debrid to unrestrict your links and download them.

Information:
- Torrents / Magnet links handling is not currently available on our extensions.
- Safari extension is not currently available for version 13 and later.

## I'm getting a slow download speed
Please make a speed test on our dedicated page (from the same device you use our service on): [https://real-debrid.com/speedtest](https://real-debrid.com/speedtest). Only make a speed test from a different device to determine if the issue is within your network / from your device.

**- If you are on Wifi:**
Prefer 5Ghz band over 2.4Ghz band for high speeds. As far as possible, connect an Ethernet cable to your device (even temporarily for diagnostic purposes) and make a new speed test to compare results. A lot of issues come from a bad Wifi connection.

**- Speed issue with VPN usage:**
When you use a VPN, your connection is redirected to a remote server. If you're getting a low download speed, try to change your VPN server location to a closer one.

**- Speed issue without VPN usage:**
Please try to restart your router / Internet box.

**- If the speed on (Multi) is at least 2x greater than the normal speed:**
You are probably on Windows. Try the solution below. Detail: Windows is not increasing the TCP connection buffer correctly and single connections are limited (most times near 3MB/s / 30Mbps).

**- If you are using Windows 7/8/8.1 or 10:**
It is important to verify that your current network connection (at the bottom right) is set as a private connection and not public. Windows limits the performance of public connections.

If your network connection is already set up as private, you can try to input these two commands inside the Windows command prompt run in Administrator to restore original performances:
```
netsh int tcp set glo autotuninglevel=disabled
netsh int tcp set glo autotuninglevel=normal
```
**- If your device (Computer / Phone / Tablet / NAS) is quite old (before ~2012) or low-end:**
It is important to verify that your device has the AES instruction set by looking at the processor specifications of the device you are using: [https://en.wikipedia.org/wiki/AES_instruction_set](https://en.wikipedia.org/wiki/AES_instruction_set). Your processor needs more power to decipher the stream without this AES instruction set, which can cause your speed to be severely limited. One of the solutions to get back full speed is to disable secure downloads in "My Account" (Warning: Your downloads may then be spied on).

If none of these solutions helped you, please contact us with the results you got from our speed test page.

## I'm getting an error when I try to download a file
Before contacting us, please check the status of the concerned hoster on Real-Debrid (you must be logged in): [https://real-debrid.com/compare](https://real-debrid.com/compare)

- A red thumb is displayed: The hoster is currently unavailable on Real-Debrid. **You don't need to contact us.** We are aware of this issue and are working to resolve it as soon as possible.
- "Not Supported" is displayed: This hoster is not supported on Real-Debrid. **You don't need to contact us, your ticket will not be processed.**
- A green thumb is displayed: This hoster should be available. You should be able to download a file from the concerned hoster. If this is not the case, please contact us by clicking on the button below.

## I would like to use a VPN on Real-Debrid
**How it works:**
By default, all server IP addresses are blocked on Real-Debrid to prevent abuse. VPN services use servers to operate, and they can be affected by these restrictions.

If you don't have a VPN yet and want to use one, we recommend choosing one of the "cooperative VPNs": [https://real-debrid.com/vpn](https://real-debrid.com/vpn). Cooperative VPN providers give us their IP addresses so we can add them to our whitelist without any action from you.

**How to know if your VPN IP address is blocked on Real-Debrid:**
Once connected through your VPN, please visit this page: [https://real-debrid.com/vpn](https://real-debrid.com/vpn)

- If you see a red message with "Your ISP or VPN IP address is currently blocked on our site ...", it means that your VPN IP address is currently blocked. Please contact us.
- Also, if you're getting the error "ip_not_allowed_X.X.X.X" when you are unrestricting a link, your VPN IP address is currently blocked. Please contact us.

**/!\\ PLEASE READ /!\\**
In these two cases, you have to provide us with a PROOF that you'll be able to send us with your message.
PROOF = screenshot of the VPN software you're using, showing the blocked IP address (a screenshot of our website is not proof).

**Information:**
About personal servers (RDP, personal VPN, private proxy, etc.), please type "Remote Traffic" in the search box above for more details.

## I would like to get more details about the torrent part
[https://real-debrid.com/torrents](https://real-debrid.com/torrents)

The torrent part allows you to convert a .torrent file or a magnet link to a Real-Debrid link. Once the conversion is completed, you will have to unrestrict the Real-Debrid link on this page to be able to download it: [https://real-debrid.com/downloader](https://real-debrid.com/downloader)

Please note that we do not provide any torrent or magnet link. Google and online indexers are your friends.

**Limits:**

- Maximal torrent size: 2000 GB
- Maximal torrent download duration: 72 hours

A Real-Debrid link remains valid for 30 days (without any uptime guarantee). After this deadline, you will no longer be able to download it.

## I would like to use Real-Debrid with my own server (Remote Upload)
If you want to download with or through a dedicated server (RDP, personal VPN, private proxy, etc.), you have to use your "Remote Traffic".

The Remote traffic available on our website also allows you to use our links on the hosters that accept the Remote Upload. The hosters will be able to download and directly save the file on their servers without using your own connection.

You can know how much Remote traffic you have in the "Limits" part

 on the right of this page: [https://real-debrid.com/downloader](https://real-debrid.com/downloader)

**How to use your Remote traffic:**
To use your Remote Traffic, you have to check the "Remote Traffic" checkbox on this page: [https://real-debrid.com/downloader](https://real-debrid.com/downloader) before you unrestrict a link. If you get the error "traffic_exhausted", it means that you have no more Remote Traffic.

If you want to buy some Remote traffic, you can do this here: [https://real-debrid.com/trafficshare](https://real-debrid.com/trafficshare). Please note that your account must be Premium to use this traffic.

## I would like to know how to use Real-Debrid
On Real-Debrid, there are two main functionalities.

**- The "Download" part:**
This feature aims to remove the restrictions (quota, download speed, etc.) of file hosters.

**How it works:**
1. Go to this page: [https://real-debrid.com/downloader](https://real-debrid.com/downloader)
2. Paste the links which are supported by Real-Debrid in the space provided for this purpose (example: [https://vimeo.com/358854871](https://vimeo.com/358854871))
3. Click on the "Unrestrict my links" button

You can find the list of supported hosts here: [https://real-debrid.com/compare](https://real-debrid.com/compare)

**- The "Torrent" part:**
This feature converts a torrent to a direct download link.

**How it works:**
1. Go to this page: [https://real-debrid.com/torrents](https://real-debrid.com/torrents)
2. Add a .torrent file or a magnet link in the space provided for this purpose, then click on the "Convert" button
3. A new window appears. Select the files you want, then click on the "Start my torrent" button
4. Real-Debrid now takes care of converting your torrent. Once the conversion is completed, one or more Real-Debrid "torrent" links will appear automatically.
5. To start the download, you have to click on the icon representing a green box on the right of the link(s) displayed.

**Important Information:**
In no way does Real-Debrid provide links, torrents, or magnet links to ANY content. We are not a content provider and will never be. You are legally responsible for links, torrents, or magnet links that you or a third party application sends to our service.

## I want to renew / pay my account
You can find our premium offers here: [https://real-debrid.com/premium](https://real-debrid.com/premium)

**Informations:**
- There are no recurring payments (one-time payment only).
- Days are cumulative, it means that if there are remaining days on your account, the days you pay will be added to your existing days.

## I want to know if my account is Premium
To verify this, you must first be logged into your account.

You can find the end date of your subscription in the website header and on your account page here: [https://real-debrid.com/account](https://real-debrid.com/account). Go to the "Account informations" part > Valid until

You can also find your payments history here: [https://real-debrid.com/payments-history](https://real-debrid.com/payments-history)

## I want to change the email address of my account
Only administrators can change an account's email address at the moment.

If you want to change the email address of your account, please contact us by clicking on the button below.

**/!\\ We're not accepting disposable / trash email addresses.**

## I have a question about the fidelity points
- You can earn fidelity points by buying Premium days.
- You can use your points when you want but you have to reach 1000 points to get free Premium days.
- 1000 fidelity points bring 30 premium days.
- You cannot send fidelity points to another member.

**How to use your fidelity points:**
You have to go to the "My Account" page ([https://real-debrid.com/account](https://real-debrid.com/account)). Then, in the "Account Settings" part, find "Convert 1000 Fidelity Points to 30 Days: Convert".

## I want to get more details about your affiliate program
You can get a bonus of 5 premium days + 50 fidelity points by inviting a friend on Real-Debrid.

To do this, please follow the instructions below:
1. Copy your affiliate link. You will find it here: [https://real-debrid.com/affiliate](https://real-debrid.com/affiliate)
2. To make the sponsorship work, your friend has to go to Real-Debrid by clicking on the link before the payment.

Information:
- The 15 days offer and reseller vouchers do not belong to the sponsorship program.
- The reseller payments do not belong to the sponsorship program.
- The bonus can't be applied after a payment.

## I want to delete my Real-Debrid account
First of all, please note that we can't delete an account with a premium state.

To check that, you need to be connected to your account.

You can find the end date of your subscription in the website header and on your account page here: [https://real-debrid.com/account](https://real-debrid.com/account). Go to the "Account informations" part > Valid until.

The payments on Real-Debrid are not recurring, just let your subscription run out and contact us after that.

If your account is no longer premium, please contact us by clicking on the button below.

## I would like to know the maximum number of devices I can use with my account
You can use your account from any public IP address, but you can't use your account from more than one public IP address at the same time. If your devices are connected through the same router, they are getting only one public IP address and you can use as many devices as you want.

If you want to use Real-Debrid from two different IP addresses at a time, you have to sign up and pay for two different accounts (with a different username and a different email address).

If you want to disable or verify a previously allowed device you can do this right here: [https://real-debrid.com/devices](https://real-debrid.com/devices) (you must be logged into your account).

## I received a warning
You received a warning by email with a password change?

**A warning is not a sanction. It's here to secure your account and remind you of the rules.**

Please note that accesses given to third-party applications (plugin, etc.) have been revoked when you received the warning for security reasons. You can manage your devices on this page: [https://real-debrid.com/devices](https://real-debrid.com/devices) (you must be connected to your account to see this page).

You can use your account from any public IP address but you can't use your account from more than one public IP address at the same time. If your devices are connected through the same router, they are getting only one public IP address and you can use as many devices as you want.

If you want to use Real-Debrid from two different IP addresses at a time, you have to sign up and pay for two different accounts (with a different username and a different email address).

## My account is suspended and I want to get more details
Your account is suspended and you want more details about that?

Please contact us by clicking on the button below.

## I would like to change my password
To change your password, first, you need to be connected to your account. Once you're connected, you can change your password from the "My account" page: [https://real-debrid.com/account](https://real-debrid.com/account)

If you're having problems with that, please contact us by clicking on the button below.

## I forgot / lost my password
If you forgot your password, please use the associated function on our website: [https://real-debrid.com/forgot](https://real-debrid.com/forgot). Your username will be in the email that will be sent to you. (Dear your_username)

If after a password change, you still can't log in, please contact us by clicking on the button below.

## I forgot my username
If you forgot your username, please use the associated function "Forgot my password" on our website: [https://real-debrid.com/forgot](https://real-debrid.com/forgot). Your username will be in the email that will be sent to you. (Dear **your_username**)

If you're having problems with that, please contact us by clicking on the button below.

## I'm not receiving any Real-Debrid emails
First, please check the SPAM folder of your email address.

Remind: our emails are sent from the address "noreply@real-debrid.com".

If you're having an issue with that, please contact us by clicking on the button below. Please fill in another email address in the next step so we can notify you when you'll get a reply on the ticket.

## I would like to get more details about the two-factor authentication
Two-factor authentication aims to protect your account from hacking. Upon successful login to your account with your username / password from an unknown location, our website automatically asks you for a temporary code.

**- If the two-factor authentication is enabled with your email address (by default):**

A temporary code is sent to the email address of your Real-Debrid account to authenticate you. Please note that the temporary code sent by email is valid for two hours only. After this deadline, a new temporary code

 will be created and sent to your email if needed.

Sent from: noreply@real-debrid.com
Email subject: "REAL-DEBRID - Sign in attempt, is it you? (2FA)"

**- If the two-factor authentication is enabled with the Google Authenticator app:**

In this case, you need to open the Google Authenticator app and type in the temporary code displayed. Please note that the Google Authenticator temporary code is valid for 30 seconds only. After this deadline, you'll need to open the Google Authenticator app to get a new temporary code.

If you didn't receive your temporary code by email or if you haven't access to the Google Authenticator app, please contact us by clicking on the button below.

## I want to change my username
Unfortunately, you can't change or modify your username on Real-Debrid.

## I cannot register on Real-Debrid
When a registration attempt fails, a red message appears below the registration form. Please take the time to read this error message and do the changes requested to validate your registration.

**Error messages you may get during registration:**

- "Special Characters are not allowed in your username (a-z, A-Z, 0-9 only)": Only the characters from "a" to "z" (lower and uppercase) and numbers from 0 to 9 can be used. Spaces are not allowed.
- "Your username is already used by one of our members.": The username you want to register with is already used by another user. In this case, you have to choose another one.
- "Your email is already used by one of our members.": It means that an account has already been created on our website with this email address. Please use the associated function "Forgot my password" on our website if you have forgotten your password: [https://real-debrid.com/forgot](https://real-debrid.com/forgot). Your username will also be displayed in the email you will receive. (Hello <your_username>)

If despite everything, you're having problems with that, please contact us by clicking on the button below.

## I can't log in to my Real-Debrid account
If you forgot your password or you can't log in, please use the associated function on our website: [https://real-debrid.com/forgot](https://real-debrid.com/forgot). Your username will be in the email that will be sent to you. (Dear **your_username**)

If after a password change, you still can't log in, please contact us by clicking on the button below.

## I have no Real-Debrid links / I get bad quality links on my device
If your app results are not satisfying, the best thing is to contact the developer of it or use another one because you are the one feeding links or torrents to Real-Debrid.

**In no way does Real-Debrid provide links, torrents, or magnet links to ANY content.**
**We are not a content provider and will never be.**

Reminder: You are legally responsible for links, torrents, or magnet links that you or a third-party application sends to our service.

Once logged into your account, the third-party application you use is sending .torrent files, magnet links, or filelocker links to our service.

- In the case of a .torrent file or a magnet link, Real-Debrid converts this into a direct download link.
- In the case of a file host link, Real-Debrid removes the restrictions (quota, download speed, etc.).

## I sent an incorrect Bitcoin / Ethereum / Litecoin amount (over/under-payment)
If you made an over-payment or under-payment, we need your wallet address so that we can refund you.

In case of over-payment, we refund the extra amount paid. In case of under-payment, we refund the whole transaction. You will have to make a new transaction to subscribe (you should not deduct fees from your payments!).

## I'm getting an error on the third-party app that I'm using with Real-Debrid
You're getting "need resolve" or "cannot make a new request because the previous response is still open: please call response.close" error.

This issue has nothing to do with Real-Debrid so we can't help you.

Please try to clean the cache of the third-party app you're using and re-authorize.

The Real-Debrid authorization code is given to you by the third-party application you are trying to connect your account to. Authorization codes change every 2 minutes and are different for each application that you want to authorize. Therefore, they are not reusable or supplied after payment.

- To authorize a third-party application on your account, please go here: [https://real-debrid.com/device](https://real-debrid.com/device)
- To view or delete authorizations previously made on your account, please go here: [https://real-debrid.com/devices](https://real-debrid.com/devices)

## I didn't sign up from the correct IP address / ISP
The IP address you had when you signed up doesn't matter. A Real-Debrid account is not linked to an IP address.

**Reminder:**
You can use your account from any public IP address but you can't use your account from more than one public IP address at the same time. If your devices are connected through the same router, they are getting only one public IP address and you can use as many devices as you want.

## My downloads are corrupted
If your downloads are corrupted and/or your archive files are not extractible, it is first necessary to check that the files are not corrupted from their source by downloading, when possible, the same files directly in free mode on the hoster.

If it is not possible or the problem persists, please contact us by providing the most useful information you have (connection type, etc.).

## How to use Real-Debrid in Synology Download Station
The .host file for Download Station can be downloaded here: [https://fcdn.real-debrid.com/plugins/realdebrid_host_synology_0_4.zip](https://fcdn.real-debrid.com/plugins/realdebrid_host_synology_0_4.zip) (Last hosts update: 2023-11-30)

Do not forget to uncheck the original "RealDebrid" in the host list. Put your logins in RealDebrid Official and you are all set.

![Download Station](https://i.imgur.com/klSVVcq.png)

**Special case: DSM 7 Beta**
Custom .host files can't be added at the moment (bug?). An alternative is to log in to your NAS IP Address in SSH (with an Administrator account):
```
ssh admin@192.168.1.X
```
You can use PuTTY ([https://www.putty.org](https://www.putty.org)) if you are using Windows.

Once logged in, type these commands:
```
sudo su
cd /tmp/ && wget https://fcdn.real-debrid.com/plugins/realdebridofficial_synology_0_4.tar.gz && tar xvf realdebridofficial_synology_0_4.tar.gz && rm -rf realdebridofficial_synology_0_4.tar.gz
```

## I would like to get a receipt about my payment
First, you need to be logged into your Real-Debrid account. You can then find your receipts through your payments history here: [https://real-debrid.com/payments-history](https://real-debrid.com/payments-history)

Please note that you can find this link in the "My account" page: [https://real-debrid.com/account](https://real-debrid.com/account)

## The download of my magnet link doesn't start
If the download of your magnet link doesn't start, please contact us.

## The download of my .torrent file doesn't start
If the download of your .torrent file doesn't start, please contact us.

## Where is the forum?
The forum is closed and no longer accessible.

## I would like to develop an app / script using your service
You can find all the information about our API at [https://api.real-debrid.com/](https://api.real-debrid.com/)

## I would like to get a client ID for my application / script
Please check the "Opensource Apps" part, you will find a Client ID you can use: [https://api.real-debrid.com/](https://api.real-debrid.com/)

## My account has not been credited after a Paysafecard payment
First, please check that your payment has been debited from your Paysafecard. To do this, please go to: [https://www.paysafecard.com/en-gb/check-balance/](https://www.paysafecard.com/en-gb/check-balance/)

Second, please check your account payments history on your Real-Debrid account. To do this, please go to: [https://real-debrid.com/payments-history](https://real-debrid.com/payments-history)

After checking, if you think that the payment has been charged and not credited on your Real-Debrid account, please contact us.

## What is the name I will be seeing on my bank statement?
Payments are debited by the name REAL DEBRID (Levallois) on the bank statement.

## 429 Error
Your browser or app is making too many requests to our service, and your requests have been blocked to protect the service from being overwhelmed.

If your app "auto-resolves" every link found over the web, you have to disable that functionality.

## I would like to get my private API token
You can get it at this address: [https://real-debrid.com/apitoken](https://real-debrid.com/apitoken)

We remind you that this token should not be shared or used in public applications because it gives access to all API features, including actions on your account. It is perfect for

 local applications or your personal scripts.

## How to use my HTTP folder into Kodi
From the home screen of Kodi, click on "Videos", then "Files":

![Kodi 1](https://fcdn.real-debrid.com/0819/images/support/http_kodi_1.png)

Click on "Add videos...":

![Kodi 2](https://fcdn.real-debrid.com/0819/images/support/http_kodi_2.png)

Click on "None", then add your HTTP folder link. You can find it on the "My Account" page: [https://real-debrid.com/account](https://real-debrid.com/account)

![Kodi 3](https://fcdn.real-debrid.com/0819/images/support/http_kodi_3.png)

Change the name to "Real-Debrid" for better viewing. You can now access your links and torrents without any addon:

![Kodi 4](https://fcdn.real-debrid.com/0819/images/support/http_kodi_4.png)

## My video streaming stops after 2 to 4 minutes while using NordVPN
NordVPN recently updated their app, and the WireGuard protocol from their server is buggy (also called NordLynx). Browse inside the NordVPN app and change your VPN setting from "Recommended" to "OpenVPN" (TCP or UDP). It will instantly fix your issues.

## I'm getting "fair_usage_limit" error
Your usage of the concerned hoster and/or from a server has been limited because it is considered abusive.

You may be able to download a few files, but you will be able to download any file the next day. If you are using a server, please use your ISP connection instead to download more or buy "Remote" traffic here: [https://real-debrid.com/trafficshare](https://real-debrid.com/trafficshare)

As a reminder, here is a copy of our terms: [https://real-debrid.com/terms](https://real-debrid.com/terms)
*Hosts without clearly indicated limits are considered unlimited, within the limit of a reasonable use of the User (based on the price and the original limitations of the host compared to the price paid on our service), any consumption beyond this so-called reasonable use may be temporarily limited.*

**Example:**
Rapidgator costs $14.99 with a limit of 1000GB for 30 days, which is about 33.3GB per day. Based on our price, you should be limited around 10GB per day. You can check your current usage on this page: [https://real-debrid.com/traffic](https://real-debrid.com/traffic). Most of the time we allow up to 10x that supposed limit.

## I'm getting "infringing_torrent" error when I add a magnet / torrent
Our service received a DMCA letter about the magnet/torrent you are trying to add. This torrent can't be downloaded anymore, and all copies you may have must be deleted for copyright compliance.

## I can't pay, it is saying I'm using "Cloudflare Wrap"
Most times it is because you are using an Apple device with the "iCloud Private Relay" functionality enabled.

To be allowed to pay by credit card, you must temporarily disable this functionality. You can find help on Apple's website: [https://support.apple.com/en-sg/guide/iphone/iph499d287c2/ios](https://support.apple.com/en-sg/guide/iphone/iph499d287c2/ios)

## My IP address is blocked but I don't use any VPN / Server
**How it works:**
By default, all server IP addresses are blocked on Real-Debrid to prevent abuse. VPN services use servers to operate, and they can be affected by these restrictions.

**How to know if your IP address is blocked on Real-Debrid:**
Please visit this page: [https://real-debrid.com/vpn](https://real-debrid.com/vpn)

- If you see a red message with "Your ISP or VPN IP address is currently blocked on our site ..."
- If you're getting the error "ip_not_allowed_X.X.X.X" when you are unrestricting a link.

**Why am I blocked if I do not use any VPN / Proxy / Server:**
Due to "IPv4 Run Out", many Internet Providers (of all kinds) are selling/buying/leasing IP addresses from established providers having way more IP addresses than they need. Our list of IP addresses is often updated, but it could be updated daily. That's how your local ISP IP address could be blocked on our website (for example, by buying/leasing an IP block from a cloud provider which was previously blocked on our website).

Please contact us to solve the problem.

**Comment by vikybak** [comment]
I received a warning via email from real debrid yesterday for sharing my Rd account with another IP address. The truth is that the router disconnected 3 or 4 times in 30 minutes while I was watching a tv show. Real debrid does not allow sending explanatory email. What should I do?

**Comment by TheKfcBandit** [comment]
> contact us by clicking on the button below.

Where is the button?
